app/models/game.rb

```diff
@@ -1,4 +1,72 @@
 class Game < ActiveRecord::Base
-
 	belongs_to :user
```

Comment:
:white_check_mark: Done @mscoutermarsh I also added a `has_many :pieces` relationship here. Without it, all my tests failed since `populate_board!` now uses `self.id`.